projects/aca-playwright-shared/src/page-objects/components/dialogs/content-node-selector-dialog.ts

```diff
@@ -36,6 +36,7 @@ export class ContentNodeSelectorDialog extends BaseComponent {
   public actionButton = this.getChild('[data-automation-id="content-node-selector-actions-choose"]');
   public locationDropDown = this.getChild('[id="site-dropdown-container"]');
   private selectedRow = this.getChild('.adf-is-selected');
+  public searchField = this.getChild('#searchInput');
```

Comment:
looks like this field is unused, please remove it

Reply:
yes, removed this changes, it was not used in current code